include/RED4ext/DynArray.hpp

```diff
@@ -181,14 +181,14 @@ struct DynArray
 
     void Reserve(uint32_t aCount)
     {
-        constexpr uint32_t alignment = std::max(8ull, alignof(T));
+        constexpr uint32_t alignment = alignof(T);
```

Comment:
Will the be OK for types with alignment < 8? I remember I only saw alignment >= 8.

Reply:
It's still max(8, a) but with ternary operator down the code :) I changed it because MINMAX macros enabled by default.

Reply:
Ah, I see. Thanks!